mne_bids/read.py

```diff
@@ -73,6 +73,30 @@ def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
+def _handle_info_reading(sidecar_fname, raw):
+    """Read associated sidecar.json and populate raw.
+
+    Handle PowerLineFrequency of recording.
+    """
+    with open(sidecar_fname, "r") as f:
+        sidecar_json = json.load(f)
+
+    line_freq = sidecar_json["PowerLineFrequency"]
```

Comment:
PowerLineFrequency is *required* in the JSON files, so the `mydict[mykey]` syntax should be safe to do (i.e., it should never result in a `KeyError`) ... however, using the `mydict.get(mykey)` syntax would be much more robust and set `line_freq` to `None` if the key was miraculously missing.